app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,26 @@
+# frozen_string_literal: true
+
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+
+  def new; end
+
+  def create
+    @comment = current_user.comments.build(comment_params)
+    redirect_to posts_path if @comment.save
+  end
+
+  def destroy
+    @comment = Comment.find_by(id: params[:id])
```

Comment:
what happens if it fails to delete? Maybe you can address this and test for it as well.
[OPTIONAL]

Same applies for likes